networking/voting_handlers.go

```diff
@@ -549,7 +556,12 @@ func get(_msg *C.struct_msg_t, _conn *C.struct_msgnetwork_conn_t, _ unsafe.Point
 		return
 	}
 
-	containerID, _ := ids.ToID(msg.Get(ContainerID).([]byte))
+	containerID, err := ids.ToID(msg.Get(ContainerID).([]byte))
+	VotingNet.log.Verbo("received get from %v\nchainID: %v\nrequestID: %v\ncontainerID: %v", validatorID, chainID, requestID, ContainerID)
+	if err != nil {
+		VotingNet.log.Debug("error parsing ContainerID: %s", err)
+		return
+	}
```

Comment:
This should assert that there is no error here.

Reply:
Why assert and not just log + return?

Reply:
Because the error should never happen. Which was why it wasn't checked before. The message parsing should ensure that this will pass.